mappings/net/minecraft/block/BucketInteractable.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS bgx net/minecraft/block/BucketInteractable
+	METHOD b getBucketFluid (Lbbq;Let;Lbpm;)Lcfc;
```

Comment:
I think `interactWithBucket` might be a better name for this as it's not only a getter. Thoughts?

Reply:
Actually, it's not a complete impl of what I would consider a method called `interactWithBucket` (it's mostly a getter).